aiohttp_sse/__init__.py

```diff
@@ -103,8 +108,14 @@ def __init__(self, *, status=200, reason=None, headers=None):
         """
         if self._ping_task is None:
             raise RuntimeError('Response is not started')
-        with contextlib.suppress(asyncio.CancelledError):
+
+        try:
             await self._ping_task
+        except asyncio.CancelledError:
+            # Re-raise if client disconnect
+            request = getattr(self, '_req', None)
+            if request is not None and request.protocol._connection_lost:
```

Comment:
Unfortunately `_connection_lost` is not guaranteed to be present in all event loop implementation as result we can not use it.

Reply:
Changed on next commit